js/reorder/distanceFunctions/tnw.js

```diff
@@ -36,7 +33,7 @@ function getTNWScore(sim1, sim2) {
     for (let i = 1; i < simulation1.length + 1; i++) {
         for (let j = 1; j < simulation2.length + 1; j++) {
             let arrowImage = 'd.png'
-            let match = matrix[i - 1][j - 1].cellMax + MATCH_SCORE + offsetPenalty(simulation1[i], simulation2[j], MAX_OFFSET_PENALTY)
```

Comment:
This was a bug that got fixed in earlier branches. This doesn't solve the "temporal" part of the algo that we talked about on 9/3